frontend/shared/src/components/wizard/Wizard.tsx

```diff
@@ -117,7 +120,13 @@ const Wizard: React.FC<WizardProps> = React.memo(
     return (
       <WizardContext.Provider value={wizardValue}>
         {header}
-        {activeStepContent}
+        {isLoading ? (
```

Comment:
`waitForLoadingCompleted` util function will now also wait for wizard transitions in tests. This will reduce `act`-errors a lot.